Documentation/Upgrade/rook-upgrade.md

```diff
@@ -143,7 +144,7 @@ by the Operator. Also update the Custom Resource Definitions (CRDs).
 Get the latest common resources manifests that contain the latest changes.
 
 ```console
-git clone --single-branch --depth=1 --branch v1.10.0-beta.0 https://github.com/rook/rook.git
+git clone --single-branch --depth=1 --branch release-1.10 https://github.com/rook/rook.git
 cd rook/deploy/examples
 ```
```

Comment:
@travisn I think we want the branch here and not the version tag, right?

Reply:
Either the tag or branch name works with the `--branch` param, so we go with the version tag in the release branch to match the release instead of the head of the release branch.